crates/orchestrator/src/jobs/mod.rs

```diff
@@ -209,8 +209,8 @@ pub async fn process_job(id: Uuid, config: Arc<Config>) -> Result<(), JobError>
     match job.status {
         // we only want to process jobs that are in the created or verification failed state.
         // verification failed state means that the previous processing failed and we want to retry
-        JobStatus::Created | JobStatus::VerificationFailed => {
-            tracing::info!(job_id = ?id, status = ?job.status, "Job status is Created or VerificationFailed, proceeding with processing");
+        JobStatus::Created | JobStatus::VerificationFailed | JobStatus::RetryAttempt => {
+            tracing::info!(job_id = ?id, status = ?job.status, "Job status is Created or VerificationFailed or RetryAttempt, proceeding with processing");
```

Comment:
I think the message here is a lil too much, since we are already logging the `status` so we can change the message to something more useful.
WDYT @Mohiiit ?